mongox/models.py

```diff
@@ -275,7 +275,7 @@ def __new__(mcs, name, bases, namespace, **kwargs):
         cls = super().__new__(mcs, name, bases, namespace)
 
         if kwargs:
-            cls.Meta = Meta
+            cls.Meta = Meta()
```

Comment:
Thanks for the PR. It would be great to have a test case for it. I think there's an example in the issue.